### PR TITLE
Improvement of the Chromium-based detections

### PR DESCRIPTION
**Pull request by Wicker25**
#### What I'm doing
Improvement of the Chromium-based detections.

### CODE REVIEW

File: src/desktop/brave.js [hunk 1]
```diff
@@ -9,45 +21,23 @@ export function detectBrave() {
 
   // Brave is based on Chromium
   var chromiumFeatures = detectChromium();
+
   if (!chromiumFeatures) {
     return;
   }
 
-  // Brave does not support the Native Client plugin (internal-nacl-plugin)
-  if ($.hasFeature('navigator.plugins.Native Client')) {
```
Comment (Wicker25) [inline src/desktop/brave.js]: I think we don't need this check anymore, due to the new detection order.

Comment (ohpe) [reply src/desktop/brave.js]: That might be the case, but I'd prefer to enforce the check.
Maybe we can check that the supported plugins are exactly the names we know:
```
$["window"]["navigator"]["plugins"]["Chrome PDF Plugin"]
$["window"]["navigator"]["plugins"]["Chrome PDF Viewer"]
```

Comment (ohpe) [reply src/desktop/brave.js]:
```js  
if (plugins.length===2 && !($.hasPlugin('Chrome PDF Plugin') && $.hasPlugin('Chrome PDF Viewer'))) {
return;
}
```

Comment (Wicker25) [reply src/desktop/brave.js]: It sounds good to me :+1:

**Pull request merged**